File: Recursion/print_name.c
#include <stdio.h>

void printName() {
    printf("Your Name\n");
}

void printNameRecursively(int n) {
    if (n > 0) {
        printNameRecursively(n - 1);
        printName();
    }
}

int main() {
    int repetitions;
    printf("Enter the number of times to print the name: ");
    scanf("%d", &repetitions);
    printNameRecursively(repetitions);
    return 0;
}

File: Recursion/print_name.py
def print_name():
    print("Your Name")

def print_name_recursively(n):
    if n > 0:
        print_name_recursively(n - 1)
        print_name()

repetitions = int(input("Enter the number of times to print the name: "))
print_name_recursively(repetitions)
